### ISSUE TITLE
Very strange behavior of the gantt plugin

### ISSUE THREAD

**Issue by Ingenu1ty** [saltcorn/saltcorn#2579]
The [svelte-gantt](https://github.com/saltcorn/svelte-gantt) plugin does something really unexpected...
What is the reason for this crazy "growth"?:

[ganttvid.webm](https://github.com/user-attachments/assets/becb493c-491b-4f47-b8ee-908e167bc442)

table configuration:

![Cnft](https://github.com/user-attachments/assets/5fa14d3f-f432-451d-add7-d84ade2df29e)

table data:

![Tdata](https://github.com/user-attachments/assets/4b8c5e47-5f63-49ae-955b-ad89faef6f15)

configuration object:
`{ "exttable_name": null, "viewname": "Gantt test view 2", "title_field": "title", "title_formula": null, "description_field": "descr", "description_formula": null, "row_field": "id", "row_label_formula": "", "dependency_table": "", "move_between_rows": true, "move_within_row": true, "start_field": "start_d", "end_field": "", "duration_field": "duration", "duration_units": "Days", "align_to_start": false, "color_field": "", "milestone_field": "", "completed_field": "", "hide_empty_rows": false, "text_color": "#000000", "edit_view": "", "task_detail_view": "", "add_on_row": false, "add_task_top": false, "show_current_time": false, "show_weekend": false, "lock_editing_switch": false, "row_height": 52, "table_width": 240, "row_padding": 6, "row_order_field": "", "row_order_descending": false, "tree_field": "", "reflectOnParentRows": false, "focus_button": false, "resource_field": "", "id": 7, "table_id": 4 }` 

theme: any-bootstrap-theme

P.S. it would also be great to update the plugin...The [underlying library](https://github.com/ANovokmet/svelte-gantt) has added **a lot of interesting features since then**...(especially [layout](https://anovokmet.github.io/svelte-gantt/docs/options/layout))

**Comment by glutamate** [comment]
Interesting (cc @atomic3930 who also reported this) - does not happen on Firefox. and we haven't changed the saltcorn plugin for 1 year.. make me thing this is some new behaviour in Chrome. We've seen a lot of strange things with recent chrome updates. Anyways i will try to update the underlying library and see if that fixes the behaviour.

**Comment by glutamate** [comment]
Yep @Ingenu1ty @atomic3930 updated the library fixed it - i have released as plugin v 0.4.0 but have not checked in detail if the update broke any other features

**Comment by Ingenu1ty** [comment]
@glutamate 
Thanks! But why are [the rows](https://anovokmet.github.io/svelte-gantt/docs/data/rows) not displayed?

![Rfscr](https://github.com/user-attachments/assets/33e47b13-2371-4bb8-ad23-06d2fc0e3927)

I think it should be something like this:

![Rowcr](https://github.com/user-attachments/assets/b70a2a5c-0ea5-452f-b9d1-0ebbbaf4c01f)

***
By the way, I really miss the ability to display an arbitrary view inside a task(or at least an arbitrary html with interpolation or "{{ mustache }}")😔...I think this feature would be useful not only for me...

**Comment by glutamate** [comment]
default colour now restored in 0.4.1

**Comment by Ingenu1ty** [comment]
> default colour now restored in 0.4.1

yeah, that's a good bugfix too, but I meant a slightly different problem:

![Sc01](https://github.com/user-attachments/assets/77864f24-4265-4248-aca8-6e31babb9c49)

Row headers, those that should be displayed on the left(and this problem same in the plugin version 3.x)... however this **bug only appears on mobile**(at least android, I haven't checked iOS) - on desktop everything is fine! 

one might think that these are chrome artifacts, but the underlying library works fine even in mobile chrome! - this suggests that the problem may be in a SC/theme css conflict or something like that...
